packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Create an API token
+excerpt: How to create a token for the Okta API
+layout: Guides
+---
+
+This page explains what an API token is, why you need one, and how to create one.
+
+**Learning outcomes**
+
+* How an Okta API token is used
+* Why it's good practice to create a service account for use with an API token
+* Alternatives to Okta API tokens
+* Token expiration
+* How API tokens are deactivated
+
+**What you need**
+
+Have an Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup).
```

Comment:
you could just use the relative URL `/signup` here, which would make the link follow more efficient, and get rid of the erroneous external link icon that the rendered link currently gets.